Update t-digest.md (#687)
RedisBloom/RedisBloom: docs/docs/t-digest.md

## Examples

> We'll demonstrate creating a t-digest with an initial compression of 100, adding items to it, and checking the estimated percentile of a value. Note that the `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default is 100. Higher values mean more accuracy. Note also that unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.

Reviewer: I don't see where you "check the estimated percentile of a value" in the example.

```suggestion
In the following example, you'll create a t-digest with an initial compression of 100, add items to it, and check the estimated percentile of a value. The `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default value is 100. Higher values mean more accuracy. Note: unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.
```

Reviewer 2: I wouldn't use "initial compression" because it suggests that it can be changed or is being changed automatically. I would use just "compression" instead. 
I also suggest changing "check the estimated percentile" to "query ...".

{{< clients-example tdigest_tutorial tdig_start >}}
> TDIGEST.CREATE bikes:sales COMPRESSION 100
OK
> TDIGEST.ADD bikes:sales 21
OK
> TDIGEST.ADD bikes:sales 150 95 75 34
OK
{{< /clients-example >}}

You can repeat calling [TDIGEST.ADD](https://redis.io/commands/tdigest.add/) whenever new observations are available

#### Estimating fractions or ranks by values

Another helpful feature in t-digest is CDF (definition of rank) which gives us the fraction of observations smaller or equal to a certain value. This command is very useful to answer questions like "*What's the percentage of observations with a value lower or equal to X*".

> >More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the **number** of observations in the sketch that are smaller than X plus half the number of observations that are equal to X, or in other words - the estimated rank of a value. The `TDIGEST.RANK` command is also variadic.

Reviewer: ```suggestion
>More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X, plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the *number* of observations in the sketch that are smaller than X, plus half the number of observations that are equal to X, or, in other words, the estimated rank of a value. The `TDIGEST.RANK` command is also variadic.
```

Reviewer 2: Please add "estimated":
> Instead of returning a fraction, it returns the ----estimated---- **number** of observations.

> The `TDIGEST.RANK` command is also variadic.
Consider using a simpler explanation. I'm not sure every reader knows what "variadic" means here.

> Let's illustrate this with an example: if we have a set of observations of people's age with gaussian distribution, we can ask a question like "What's the percentage of bike racers are younger than 50 years?"

Reviewer 2: We don't need the Gaussian distribution assumption. t-digest works well with any distribution.

Use `TDIGEST.MIN` and `TDIGEST.MAX` to retrieve the minimal and maximal values in the sketch, respectively.

{{< clients-example tdigest_tutorial tdig_min >}}
> TDIGEST.MIN racer_ages
"19.27"
> TDIGEST.MAX racer_ages
"85.709999999999994"
{{< /clients-example >}}

> Both return nan when the sketch is empty.

Reviewer 2: change nan to `nan` ?
